src/router/index.js

```diff
@@ -56,20 +56,20 @@ router.beforeEach((to, from, next) => {
   const publicPages = ["/lobby", "/playai", "/", "/contact"];
   const authRequired = !publicPages.includes(to.path);
 
+  async function verify() {
   if (authRequired) {
-    axios.get("/api/users/secret", {}).then((response) => {
+    await axios.get("/api/users/secret", {}).then((response) => {
```

Comment:
`await` is a syntax sugar to use `Promise` in more simple way.
So you don't need `.then(` here.
That can be cleaner with
```
let response = await axios.get("/api/users/secret", {})
```